src/components/PostDetails/PostDetails.jsx

```diff
@@ -1,45 +1,95 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
 import { NewCommentForm } from '../NewCommentForm';
+
 import './PostDetails.scss';
 
-export const PostDetails = () => (
-  <div className="PostDetails">
-    <h2>Post details:</h2>
+import { getPostComments, deleteComment } from '../../api/comments';
+import { getPostDetails } from '../../api/posts';
 
-    <section className="PostDetails__post">
-      <p>sunt aut facere repellat provident occaecati excepturi optio</p>
-    </section>
+export const PostDetails = ({ postId }) => {
+  const [comments, setComments] = useState([]);
+  const [postTitle, setPostTitle] = useState('');
+  const [postDetails, setPostDetails] = useState('');
```

Comment:
```suggestion
  const [postDetails, setPostDetails] = useState(0);
```
You use postId for that state.